src/deploy.js

```diff
@@ -55,7 +55,7 @@ module.exports = (customConfig) => {
         // Prepend the folder to all source files
         path.dirname = `${config.folder}/${path.dirname}`;// eslint-disable-line no-param-reassign
       })))
-      .pipe(publisher.publish())
+      .pipe(publisher.publish(config.headers))
```

Comment:
Should we also specify `headers` in `config`? And maybe set it to `{}` by default?